src/overlays/actors/ovl_En_Bal/z_en_bal.h

```diff
@@ -3,16 +3,40 @@
 
 #include "global.h"
 #include "z64snap.h"
+#include "objects/object_bal/object_bal.h"
+
+#define TINGLE_GET_MAP_ID(thisx) ((thisx)->params & 0xFF)
 
 struct EnBal;
 
 typedef void (*EnBalActionFunc)(struct EnBal*, PlayState*);
 
 typedef struct EnBal {
     /* 0x000 */ PictoActor picto;
-    /* 0x148 */ char unk_148[0x44];
+    /* 0x148 */ SkelAnime skelAnime;
     /* 0x18C */ EnBalActionFunc actionFunc;
-    /* 0x190 */ char unk_190[0x224];
-} EnBal; // size = 0x3B4
+    /* 0x190 */ ColliderJntSph collider;
+    /* 0x1B0 */ ColliderJntSphElement colliderElement;
```

Comment:
```suggestion
    /* 0x1B0 */ ColliderJntSphElement colliderElements[1];
```
Even though it only has 1 element, it still seems appropriate to make it an array (also seems to be the common way to express it in the code base)